style/patch.ts

```diff
@@ -1,23 +1,29 @@
-import { expandGlobSync } from "https://deno.land/std@0.172.0/fs/mod.ts";
+import $ from "https://deno.land/x/dax@0.27.0/mod.ts";
 import css from "npm:css";
 
+await $`rm -rf style/node_modules/@primer/primitives`;
+await $`npm install`.cwd("./style").quiet();
+
 const colorVariables = new Set<string>();
 const variableRegex = /--[\w-]+/g;
 
-for (
-  const { path } of [
-    { path: "main.scss" },
-    ...expandGlobSync("node_modules/@primer/css/markdown/*.scss"),
-  ]
-) {
-  const file = Deno.readTextFileSync(path);
+const cwd = $.path(".");
+const scssFiles = [
+  cwd.join("./style/main.scss"),
+  ...Array.from(
+    cwd.expandGlobSync("node_modules/@primer/css/markdown/*.scss"),
```

Comment:
Actually, does this match any files? Maybe it should be `cwd.expandGlobSync("styles/node_modules/@primer/css/markdown/*.scss")`?

Reply:
I think you might be right... let me look at this again.